### ISSUE TITLE
Filter is ignored

### ISSUE THREAD

**Issue by ravinggenius** [gajus/roarr-cli#1]
I'm having trouble filtering noise from my logs. Below I have two sample log entries. The second entry *should* be removed after piping to `roarr filter`, but it keeps coming through. One of my libraries (`next`) is noisy, so I added `--exclude-orphans`, and that does work as expected.

Doesn't filter the second line:

```shell
npm run start | npx roarr filter --exclude-orphans '{ "context.logLevel": { gt: 10 } }'
```

Also doesn't filter second line:

```shell
npm run start | npx roarr filter --exclude-orphans '{ "context.request.path": "/_next", _not: true, _start: true }'
```

Input:

```json
{"context":{"package":"less-longer","namespace":"server:routes","logLevel":30,"requestTime":10,"request":{"protocol":"http","http":"1.1","ip":"::1","method":"GET","original":"/l?resume=/s","path":"/l","params":{},"query":{"resume":"/s"},"headers":{"host":"localhost:3000","connection":"keep-alive","pragma":"no-cache","cache-control":"no-cache","upgrade-insecure-requests":"1","user-agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36","accept":"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8","referer":"http://localhost:3000/l?resume=/s","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.9","cookie":"io=Vmatmv_OLh7qXRVHAAAC"}},"response":{"statusCode":200,"statusMessage":"OK"}},"message":"Request complete","sequence":2,"time":1553464153412,"version":"1.0.0"}
{"context":{"package":"less-longer","namespace":"server:routes","logLevel":10,"requestTime":19,"request":{"protocol":"http","http":"1.1","ip":"::1","method":"GET","original":"/_next/static/development/pages/_app.js?ts=1553464153410","path":"/_next/static/development/pages/_app.js","params":{"0":"/_next/static/development/pages/_app.js"},"query":{"ts":"1553464153410"},"headers":{"host":"localhost:3000","connection":"keep-alive","pragma":"no-cache","cache-control":"no-cache","user-agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36","accept":"*/*","referer":"http://localhost:3000/l?resume=/s","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.9","cookie":"io=Vmatmv_OLh7qXRVHAAAC"}},"response":{"statusCode":200,"statusMessage":"OK"}},"message":"Request complete","sequence":3,"time":1553464153463,"version":"1.0.0"}
```

What am I doing wrong?

**Comment by gajus** [comment]
It is because the default context is 2 surrounding lines:

https://github.com/gajus/roarr-cli#filter-configuration

```
lodge1~/Documents/dev/gajus/roarr-cli(master|✔) % echo '{"context":{"package":"less-longer","namespace":"server:routes","logLevel":30},"message":"Request complete","sequence":2,"time":1553464153412,"version":"1.0.0"}
{"context":{"package":"less-longer","namespace":"server:routes","logLevel":10},"message":"Request complete","sequence":3,"time":1553464153463,"version":"1.0.0"}' | roarr filter '{ "context.logLevel": { gt: 20 } }' | roarr pretty-print

[2019-03-24T21:49:13.412Z] INFO (30) (@less-longer) (#server:routes): Request complete
[2019-03-24T21:49:13.463Z] TRACE (10) (@less-longer) (#server:routes): Request complete
lodge1~/Documents/dev/gajus/roarr-cli(master|✔) % echo '{"context":{"package":"less-longer","namespace":"server:routes","logLevel":30},"message":"Request complete","sequence":2,"time":1553464153412,"version":"1.0.0"}
{"context":{"package":"less-longer","namespace":"server:routes","logLevel":10},"message":"Request complete","sequence":3,"time":1553464153463,"version":"1.0.0"}' | roarr filter '{ "context.logLevel": { gt: 20 } }' --context 0 | roarr pretty-print

[2019-03-24T21:49:13.412Z] INFO (30) (@less-longer) (#server:routes): Request complete

```

tl;dr; set `--context 0`

**Comment by ravinggenius** [comment]
So basically I was holding it wrong again. Thanks for your help!

**Comment by gajus** [comment]
To be fair, this is not really intuitive.

At first I thought this is a bug and started looking at the codebase.

Off top of my head, I cannot think of any to improve the UX, though.
